src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,35 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager extends Employee {
```

Comment:
Fajnie jakby ta klaska była abstrakcyjna. W uwadze powyżej zamieściłem linijkę, w której poprawiam Cię, by metoda była abstrakcyjna. Oznacza to, że nie posiada implementacji. Implementacje muszą dostarczyć jej dzieci, czyli klasy po niej dziedziczące. Zauważ, że nasza hierarchia wygląda tak:
`Manager` - rodzic. `EmployeeNumberManager`, `BudgetManager` - dzieci.
Rodzic deklaruje metodę, dzieci implementują je na swój własny sposób.